src/naksu/log/log.go

```diff
@@ -76,8 +76,14 @@ func GetNewDebugFilename() string {
 		panic("Could not get home directory")
 	}
 
-	if existsDir(filepath.Join(homeDir, "ktp")) {
-		newDebugFilename = filepath.Join(homeDir, "ktp", "naksu_lastlog.txt")
+	ktpDir := filepath.Join(homeDir, "ktp")
+	err = os.Mkdir(ktpDir, 0700)
```

Comment:
Pitäiskö tää os.Mkdir tehdä vain jos hakemistoa ei jo ole? Nyt joka käynnistyksellä ekan jälkeen tulee terminaaliin herja:
`Warning: log.GetNewDebugFilename() could not create directory '/home/visa/ktp': mkdir /home/visa/ktp: file exists`

Reply:
Ei herranjumala :hankey: .